src/components/surveys/SurveyList.tsx

```diff
@@ -218,6 +218,7 @@ const SurveyList: React.FunctionComponent<{}> = () => {
             <StyledSurveysContainer key="container">
               {surveys?.map((survey, index) => (
                 <Link
+                  component="button"
```

Comment:
Test was failing because the `Link` doesn't include an `href` prop, so doesn't have a 'link' role. Instead, render as a button, which is recommended by [MUI for accessibility](https://mui.com/material-ui/react-link/#accessibility) when there is not a meaningful `href`.